non-free.md

```diff
@@ -103,6 +103,7 @@
 
 - [Axigen](https://www.axigen.com/mail-server/free/) `⊘ Proprietary` -  Great alternative to open source. It's a turnkey messaging solution, perfect for small & micro businesses, integration projects or test environments.
 - [Cloudron](https://cloudron.io) `⊘ Proprietary` - Open-core software allowing you to effortlessly self-host web apps on your server. ([Source Code](https://git.cloudron.io/groups/cloudron))([Demo](https://my.demo.cloudron.io/)) `Nodejs/Docker`
+- [Easypanel](https://easypanel.io) `⊘ Proprietary` - Modern server control panel powered by Docker `Nodejs/Docker`
```

Comment:
The order of your entry needs to change a bit, the language also needs to change to just `Docker`. See suggested changes below!

```suggestion
- [Easypanel](https://easypanel.io) `⊘ Proprietary` - Modern server control panel powered by Docker `Docker`
```